sys/aarch64/pmap.c

```diff
@@ -602,3 +604,25 @@ void pmap_delete(pmap_t *pmap) {
   free_asid(pmap->asid);
   pool_free(P_PMAP, pmap);
 }
+
+/*
+ * Increase usable kernel virtual address space to at least maxkvaddr.
+ * Allocate page table (level 1) if needed.
```

Comment:
Why does the function allocate level 1 pages, but not level 2 or 3?

Reply:
My mistake in comment.